Find Food Trucks SF/Find Food Trucks SF/FoodTruckDetailViewController.swift

```diff
@@ -8,10 +8,33 @@
 
 import UIKit
 
-class FoodTruckDetailViewController: UIViewController {
+class FoodTruckDetailViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    
+    @IBOutlet weak var foodItemsTableView: UITableView!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var daysHoursLabel: UILabel!
+    @IBOutlet weak var addressLabel: UILabel!
+    @IBOutlet weak var locationDescriptionLabel: UILabel!
+    
+    var foodTruck: FoodTruck!
     
     override func viewDidLoad() {
+        super.viewDidLoad()
         
+        nameLabel.text = foodTruck.name
+        daysHoursLabel.text = foodTruck.daysHours
+        addressLabel.text = foodTruck.address
+        locationDescriptionLabel.text = foodTruck.locationDescription
     }
     
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
```

Comment:
Maybe we can add images matching the menu item as a future nice feature to have.
Example burgers in menu matches the burger image.